metadoc-js/src/test/scala/metadoc/NavigationTest.scala

```diff
@@ -0,0 +1,83 @@
+package metadoc
+
+import org.scalatest.FunSuite
+import monaco.Range
+
+class NavigationTest extends FunSuite {
+  test("Navigation.parseState") {
+    val state = Navigation.currentState(None, "/path")
+    assert(state.isDefined)
+    assert(state.get.path == "/path")
+    assert(state.get.selection.isEmpty)
+
+    val stateWithSelection = Navigation.parseState("/path2#L11")
+    assert(stateWithSelection.isDefined)
+    assert(stateWithSelection.get.path == "/path2")
+    assert(stateWithSelection.get.selection == Some(Navigation.Selection(11, 1, 12, 1)))
+
+    assert(Navigation.parseState("").isEmpty)
+    assert(Navigation.parseState("#/path2#L11").isEmpty)
+  }
+
+  test("Navigation.curentState") {
+    val stateFromHash = Navigation.currentState(None, "/path")
```

Comment:
L23-26 are identical to L8-11, intentional?

Reply:
They end up testing the same thing, but at different API levels.

Reply:
Oh, I see. Fixed the above to call `Navigation.parseState`.